index.js

```diff
@@ -20,19 +28,55 @@ AFRAME.registerComponent('simple-link', {
   /**
    * Called once when component is attached. Generally for initial setup.
    */
-  init() {},
+  init() {
+    this.navigate = this.navigate.bind(this);
+    const { el } = this;
+
+    el.setAttribute('geometry', {
+      primitive: 'circle',
+      radius: 1
+    });
+    if (this.data.image) {
+      el.setAttribute('material', {
+        src:
+          typeof this.data.image === 'string'
+            ? this.data.image
+            : this.data.image.src,
+        color: this.data.color
+      });
+    }
+
+    const textEl = this.textEl || document.createElement('a-entity');
+
+    textEl.setAttribute('text', {
+      color: this.data.textColor,
+      align: 'center',
+      font: 'kelsonsans',
```

Comment:
It might make sense to have this font be configurable.

Reply:
Added